openapi/MainResources-local-openapi-v3.yml

```diff
@@ -464,31 +462,31 @@ components:
         allDoors:
           type: array
           items:
-            type: object
+            $ref: '#/components/schemas/Window'
         openDoors:
           type: array
           items:
-            type: object        
+            $ref: '#/components/schemas/Window'  
         unknownDoors:
           type: array
           items:
-            type: object        
+            $ref: '#/components/schemas/Window'
         allWindows:
```

Comment:
`allWindows` should not be available anymore

Reply:
Currently it still is
![image](https://github.com/BoschSmartHome/bosch-shc-api-docs/assets/20702171/1ec92b89-3c4a-4060-8b1b-d609c4b1d2f1)

Reply:
Oh right, I just remembered that we had removed it from the push notification that is sent to Android due to size restrictions. Thought, we removed it for the REST endpoint as well.